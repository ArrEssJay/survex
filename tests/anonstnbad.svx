*fix . 1 2 3
*fix .. 2 3 4
*fix ... 3 4 5
*fix 1 0 0 0
. ... 10.00 123 -01
... .. 0.91 359 +67
.. .   1.23 321 -21
