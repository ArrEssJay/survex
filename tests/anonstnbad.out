./anonstnbad.svx:1:6: Character "." not allowed in station name (use *SET NAMES to set allowed characters)
./anonstnbad.svx:2:6: Character "." not allowed in station name (use *SET NAMES to set allowed characters)
./anonstnbad.svx:3:6: Character "." not allowed in station name (use *SET NAMES to set allowed characters)
./anonstnbad.svx:5:6: Can't have a leg between two anonymous stations
./anonstnbad.svx:6:7: Can't have a leg between two anonymous stations
./anonstnbad.svx:7:5: Can't have a leg between two anonymous stations

Removing trailing traverses...

Concatenating traverses between nodes...

Simplifying network...

Calculating network...

Calculating traverses between nodes...

Calculating trailing traverses...
./anonstnbad.svx:4: warning: Unused fixed point "1"

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
Done.
There were 1 warning(s) and 6 error(s) - no output files produced.
